dao/src/integ-test/java/uk/gov/moj/sdt/dao/IndividualRequestDaoTest.java

```diff
@@ -114,111 +110,103 @@ public void setUp() {
+        createIndividualRequest(customerRequestReference, LocalDateTime.now().plusDays((dataRetentionPeriod + 1) * -1L));
         IIndividualRequest individualRequest = individualRequestDao.getIndividualRequest(bulkCustomer, customerRequestReference, dataRetentionPeriod);
-        Assert.assertNull(individualRequest);
-
+        assertNull(individualRequest);
     }
 
     /**
      * Test stale individual request updated 1 day ago.
      */
     @Test
-    public void testStaleIndividualRequestWithDeadLetter() {
+    void testStaleIndividualRequestWithDeadLetter() {
         // Get requests that have been updated more than 12 hrs ago.
         final List<IIndividualRequest> individualRequests = individualRequestDao.getStaleIndividualRequests(720);
 
-        Assert.assertNotNull(individualRequests);
-
-        Assert.assertEquals("The individual requests size is not correct", 2, individualRequests.size());
-
+        assertNotNull(individualRequests);
+        assertEquals(2, individualRequests.size(), "The individual requests size is not correct");
     }
 
     /**
      * Test get individual request not found.
      */
     @Test
-    public void testGetBulkSubmissionNotFound() {
-
+    void testGetBulkSubmissionNotFound() {
         final IIndividualRequest individualRequest =
                 individualRequestDao.getIndividualRequest(bulkCustomer, "NOT_FOUND", dataRetentionPeriod);
 
-        Assert.assertNull(individualRequest);
-
+        assertNull(individualRequest);
     }
 
     /**
      * Test where pending individual requests are found.
      */
     @Test
-    public void getPendingIndividualRequestsFound() {
+    void getPendingIndividualRequestsFound() {
         final List<IIndividualRequest> individualRequests = individualRequestDao.getPendingIndividualRequests(3);
 
-        Assert.assertNotNull(individualRequests);
-        Assert.assertTrue("Pending individual requests found", individualRequests.size() > 0);
-        Assert.assertEquals("The pending requests size is not correct", 1, individualRequests.size());
+        assertNotNull(individualRequests);
+        assertTrue(individualRequests.isEmpty(), "Pending individual requests found");
+        assertEquals(1, individualRequests.size(), "The pending requests size is not correct");
     }
 
     /**
      * Test where pending individual requests are not found.
      */
     @Test
-    public void getPendingIndividualRequestsNotFound() {
+    void getPendingIndividualRequestsNotFound() {
         final List<IIndividualRequest> individualRequests = individualRequestDao.getPendingIndividualRequests(4);
 
-        Assert.assertNotNull(individualRequests);
-        Assert.assertTrue("Pending individual requests not found", individualRequests.size() == 0);
+        assertNotNull(individualRequests);
+        assertTrue(individualRequests.isEmpty(), "Pending individual requests not found");
```

Comment:
May want to revert back to original code instead of isEmpty() - see previous comment.

Reply:
size == 0 and isEmpty here looks correct

Reply:
Agreed.

Just noticed that the message displayed is incorrect though.  The test isn't expecting any requests to be found, so the message will be displayed if this is false, i.e. they are found.  The message should therefore be "Pending individual requests found".

Reply:
Sorted